### PR TITLE
Add music and videos

### LINKED ISSUE #13: Add music and video resources for livestream
This is a local issue for https://github.com/publiccodenet/bumperscripter/issues/12 tracking the move of branded music and video resources to reuse in the Foundation for Public Code.

### PR DESCRIPTION
**Pull request by Ainali**
Fixes #13 and is also part of the solution for https://github.com/publiccodenet/bumperscripter/issues/12

-----
[View rendered README.md](https://github.com/publiccodenet/brand/blob/add-livestream-files/README.md)
[View rendered music/readme.md](https://github.com/publiccodenet/brand/blob/add-livestream-files/music/readme.md)
[View rendered templates/readme.md](https://github.com/publiccodenet/brand/blob/add-livestream-files/templates/readme.md)
[View rendered video/readme.md](https://github.com/publiccodenet/brand/blob/add-livestream-files/video/readme.md)

### CODE REVIEW

**Review by felixfaassen — COMMENTED** [review]
@Ainali no blocker for me but would be nice to add the license for the music as well.

**Comment by Ainali** [comment]
> @Ainali no blocker for me but would be nice to add the license for the music as well.

I believe that the files here in Brand are not freely licensed at all, they are just "© Foundation for Public Code, 2018" per the readme (where the year should perhaps be updated).

**Review by felixfaassen — COMMENTED** [review]
@Ainali great work organizing the media files. One thought I have is to maybe add a AUTHORS.md file. Reason I bring it up is that I am always sensitive about my music. I have no problems putting out in the world but it would me nice to have a place of attribution of authors artwork just as we have an authors file for the standard.

File: video/readme.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Video
+
+Here are some music files that are meant to reinforce the philosophy of the brand.
```
Comment (bvhme) [inline video/readme.md]: I'm guessing this is copy-pasted, but these aren't `music`

**Review by bvhme — CHANGES_REQUESTED** [review]
Very good, ony minor piece of documentation that needs improvement.

**Comment by Ainali** [comment]
> @Ainali great work organizing the media files. One thought I have is to maybe add a AUTHORS.md file. Reason I bring it up is that I am always sensitive about my music. I have no problems putting out in the world but it would me nice to have a place of attribution of authors artwork just as we have an authors file for the standard.

@felixfaassen  Yes, that's reasonable. Should we make that a separate PR since there will be other authors in that list not related to these files?

**Comment by felixfaassen** [comment]
> > @Ainali . Excellent idea let's make a separate pull request for that

**Review by bvhme — APPROVED** [review]
Great, I'm guessing the video and audio also needs to be versioned.

**Pull request merged**